.chloggen/tlscheck-error-handling.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: tlscheckreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Do not crash on target validation & implement better scrape errors
```

Comment:
Since this moving the feedback on invalid targets, could you adds some documentation to explain what a user needs to monitor for?

Reply:
I added some verbiage to the README explaining the change in behavior and instructing users to look at the File Stats receiver if they wish to monitor for the existence of certificate files on disk.